next.config.js

```diff
@@ -7,6 +11,70 @@ const nextConfig = {
   }
 }
 
+// API endpoints that should be forwarded to the legacy site
+const legacyAPIPaths = [
+  '/total-ogn',
+  '/total-ogv',
+  '/total-ousd',
+  '/circulating-ogn',
+  '/circulating-ogv',
+  '/social-stats',
+  '/mailing-list/:action*',
+]
+
+// Map legacy routes to new ones here
+const legacyPageMappings = [
+  // [source, destination]
+  ['/dashboard', '/'],
```

Comment:
This is going to be replaced by a redesigned dashboard. I wouldn't expect the old one to go away but I suppose it might move to the Story domain?

Reply:
I'm not sure if it can be moved over to the Story repo, since we don't use React or Node.js in the legacy repo. It will need rewriting from scratch. What's the priority on that one though?